### PR TITLE
net: mgmt: Support to disable dfs channels in scan operation

### PR DESCRIPTION
**Pull request by ajayparida**
Default scan operation includes dfs channels which are passive. A significant amount of time is taken for scan to complete. To reduce overall scan duration user can disable scanning on dfs channels.

### CODE REVIEW

File: subsys/net/l2/wifi/Kconfig [hunk 1]
```diff
@@ -72,6 +72,12 @@ config WIFI_SHELL_MAX_AP_STA
 	  This option defines the maximum number of APs and STAs that can be managed
 	  in Wi-Fi shell.
 
+config WIFI_MGMT_SCAN_DISABLE_DFS_CHANNELS
+	bool "Disables dfs channels in scan operation."
+	default n
```
Comment (pdgendt) [inline subsys/net/l2/wifi/Kconfig:77]: `default n` has to be omitted.

**Review by pdgendt — CHANGES_REQUESTED** [review]
I think part of the implementation is missing, as this `kconfig` symbols isn't used?

**Comment by ajayparida** [comment]
> I think part of the implementation is missing, as this `kconfig` symbols isn't used?

Yeah, It should be part of driver. I will do the correction

**Comment by pdgendt** [comment]
> Yeah, It should be part of driver. I will do the correction

You can keep this PR open and simply force push your additional changes.

**Comment by ajayparida** [comment]
> > Yeah, It should be part of driver. I will do the correction
> 
> You can keep this PR open and simply force push your additional changes.

There is no upstream changes for this work, as the Kconfig changes will be part of driver Kconfig. That's the reason I closed this.

**Pull request closed without merging**